en/docs/Learn/APISecurity/OAuth2DeepDive/provisioning-out-of-band-oauth-clients.md

```diff
@@ -3,50 +3,74 @@
-    ![]({{base_path}}/assets/attachments/103334858/103334848.png)
-    You have now created the OAuth client and are provided with the OAuth client key and OAuth client secret.
-    ![]({{base_path}}/assets/attachments/103334858/103334846.png)
-5.  Enable the option to provide out-of-band keys by opening the `<APIM_HOME>/repository/deployment/server/jaggeryapps/store/site/conf/site.json` file and changing the `"mapExistingAuthApps"` setting to `true` .
+    
+     <a href="../../../../../assets/img/Learn/register-oauth-app.png" ><img src="../../../../../assets/img/Learn/register-oauth-app.png" alt="Register Oauth app" 
+          title="Register Oauth app" width="50%" /></a>  
+    
+    Now you have successfully created an OAuth client and generated consumer key secret pair for it.
+   
+     <a href="../../../../../assets/img/Learn/external-oauthapp-credentials.png" ><img src="../../../../../assets/img/Learn/external-oauthapp-credentials.png" alt="Oauth app credentials" 
+          title="Oauth app credentials" width="50%" /></a> 
+    
+## Provisioning an out-of-band OAuth client in API Manager
+
+1.  Open `<APIM_HOME>/repository/conf/deployment.toml` file and add following config under `apim.devportal` config to enable the option to provide out-of-band keys. 
 
     ``` java
-        "mapExistingAuthApps" : true
+        [apim.devportal]
+        enable_key_provisioning=true
     ```
 
-        !!! note
-    Note that the the ability to provision Out-of-Band Auth client will only be available for the applications that you created **after** doing this configuration.
+    !!! Note
+        Note that the the ability to provision Out-of-Band Auth client will only be available for the applications that you created **after** applying this configuration.
+
 
+2.  Restart the server
 
-6.  Sign into the WSO2 API Store.
-`https://<Server Host>:9443/store         `
-7.  Click **Applications** .
-    ![]({{base_path}}/assets/attachments/103334858/103334853.png)
-8.  Click on the respective application to view the subscriptions details for the application.
-    ![]({{base_path}}/assets/attachments/103334858/103334845.png)
-9.  Provision an out-of-band OAuth client for the required environment.
+3.  Sign into the WSO2 API Dev Portal.
+`https://<Server Host>:9443/devportal   
+      `
+4.  Create an application following the steps given in [Create Application]({{base_path}}/Learn/ConsumeAPI/ManageApplication/create-application).
+    
+5.  Click on the respective application to view the credential details.
+    
+     <a href="../../../../../assets/img/Learn/application-select.png" ><img src="../../../../../assets/img/Learn/application-select.png" alt="App listing" 
+          title="App listing" width="50%" /></a>     
+    
+6.  Click on **Production Keys** section to provision an out-of-band OAuth client for production environment. Notice that you now see a Provide Keys section for your application. 
```

Comment:
for the production environment. You should now see a ** Provide Existing OAuth Keys ** section below the **Production Key and Secret** section.